uiService/root.go

```diff
@@ -53,8 +53,11 @@ func SetupService(cfg *config.Config) (*UIService, error) {
 	}
 
 	checkUIDistSymlink(cfg)
-	checkVersionsRoot(cfg)
```

Comment:
why did you remove the `checkVersionsRoot` call?

Reply:
I think I was planning on doing versionsRoot startup in a single method but changed my mind and forgot to put it back!